subprojects/packagefiles/theora/lib/meson.build

```diff
@@ -224,6 +225,8 @@ libtheora = library(
   c_args: compile_args,
   link_args: theora_version_flag,
   vs_module_defs: 'theora.def', # Missing in the official tarballs
+  soversion: theora_soversion,
```

Comment:
`soversion:` is redundant here (and above). If `version:` is specified, the first component is set as `soversion:`.

Reply:
Also need `darwin_versions:` set to the following:
```
libtheora.0.dylib (compatibility version 4.0.0, current version 4.10.0)
libtheoradec.1.dylib (compatibility version 3.0.0, current version 3.4.0)
libtheoraenc.1.dylib (compatibility version 3.0.0, current version 3.2.0)
```

(copying from the gstreamer issue so it's updated in both places)

Reply:
Updated accordingly (I think).